sources/zsh/history.d/volatility3

```diff
@@ -0,0 +1,15 @@
+volatility3 -f "$DUMP" windows.info
+volatility3 -f "$DUMP" windows.pslist
+volatility3 -f "$DUMP" windows.psscan
+volatility3 -f "$DUMP" windows.pstree
+volatility3 -f "$DUMP" windows.handles --pid $PID
+volatility3 -f "$DUMP" windows.dlllist --pid $PID
+volatility3 -f "$DUMP" windows.cmdline
+volatility3 -f "$DUMP" windows.netscan
+volatility3 -f "$DUMP" windows.netstat
+volatility3 -f "$DUMP" windows.registry.hivescan
+volatility3 -f "$DUMP" windows.registry.hivelist
+volatility3 -f "$DUMP" windows.registry.printkey
+volatility3 -f "$DUMP" windows.registry.printkey ‑‑key “Software\Microsoft\Windows\CurrentVersion”
+volatility3 -f "$DUMP" windows.filescan
+volatility3 -f "$DUMP" windows.malfind
```

Comment:
I think in day-to-day operations, users won't create the `$DUMP` environment variable. What do you think about replacing `$DUMP` with `memory.dmp`?

Reply:
Yes I agree, it's more intuitive !